### PR TITLE
[ MAIN PROD RELEASE ] : [ Release v1.0.0 ]

### PR DESCRIPTION
**Pull request by devilicious-captain**

### CODE REVIEW

**Review by Gaurav252317 — CHANGES_REQUESTED** [review]
Change the request

File: getData.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const connectToQueue = async () => {
   await channel.assertQueue(queue, { durable: false });
 
   channel.consume(queue, (msg) => {
-    console.log(msg, "Msg Data");
```
Comment (Gaurav252317) [inline getData.js:13]: Change this console.log

**Review by Gaurav252317 — COMMENTED** [review]
Please check the review

File: getData.js (hunk 1 above)
Comment (devilicious-captain) [reply getData.js:13]: Bypassing the message

**Review by Gaurav252317 — APPROVED** [review]
Approved

**Pull request merged**